src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Consent.cshtml

```diff
@@ -0,0 +1,78 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.ConsentModel
+@{
+}
```

Comment:
We need to add a title (that matches the page's h1), using ViewData["Title"].

(We'll need one on the previous page too.)

Reply:
Fixed